addons/gdshell/docs/en/commands/index.md

```diff
@@ -0,0 +1,33 @@
+<a href="https://github.com/Kubulambula/Godot-GDShell">
+  <img src="https://github.com/Kubulambula/Godot-GDShell/blob/main/addons/gdshell/docs/assets/logo.png" align="left" width="80" height="80">
+</a>
+
+# GDShell Command List
+
+All commands are subject to change. This article is simply a list of commands, each command has its own article explaining syntax, flags, and usage. 
+
+<!--
+  - Your README.md says you are trying to mimic the feel of BASH, so
+  - I am trying to make them in line with that, but feel free
+  - to change any you feel could be better!
+  -->
+
+Command arguments in brackets `()` are optional, arguments in `{}` are sometimes required, in `<>` are always required. This article does not exaustively list the arguments, only the basic ones. 
+
+The following are **curently implemented** in some way or another: 
+- [`gdfetch`](gdfetch.md): Prints basic project information to the terminal. 
+- [`man (command)`](man.md): Prints manual info about the command specified. (Defult value: `man man`)
+- [`echo <string>`](echo.md): Prints back the string passed in.
+- [`clear`](clear.md): Clears the terminals text. 
+- [`bool`](bool.md): Sets a boolean value in code. <!-- can you confirm? Your code isnt super clear. -->
```

Comment:
```suggestion
- [`bool`](bool.md): Sets a boolean value in code.
```

It can be used as a constant with conditional operators for debugging or as a other command's input. The command itself doesn't do anything useful